harvester/core/tests/commands/harvest_basic_content.py

```diff
@@ -60,7 +74,7 @@ def get_command_instance(self):
         command.info = lambda x: x
         return command
 
-    @patch(GET_EDUREP_OAIPMH_SEEDS_TARGET, return_value=DUMMY_SEEDS)
+    @patch(GET_EDUREP_OAIPMH_SEEDS_TARGET, return_value=DOWNLOAD_ALLOWED_DUMMY_SEEDS)
```

Comment:
Deze test is eigenlijk de e2e test voor de backend. Het brengt alle losse delen samen als het ware en test of het geheel ook werkt. Waarom neem je niet de analysis_allowed=False op in deze test? Ik zou denken dat dat goed moet gaan.